docworks-dts/lib/multiple-files/repoCreator.js

```diff
@@ -14,7 +15,7 @@ const createModulesFilesMap = ({ services, run$wFixer }) => {
 	const servicesMap = createHierarchicalServicesMap(services)
 	return Object.keys(servicesMap).reduce((filesMap, moduleName) => {
 		const rootService = servicesMap[moduleName]
-		if (isEmptyModule(rootService)) return filesMap
+		if (isEmptyModule(rootService) || isIncludesRemovedLabel(rootService)) return filesMap
```

Comment:
maybe it's better to remove deleted items inservices and subServices once when building the repo.
```
`const rootServicesWithoutDeleted = getServiceFilteredProperties(rootService)`
return {
			...filesMap,
			[moduleName]: is$w(moduleName)
				? $wGlobalNamespaceBuilder(rootServicesWithoutDeleted, { run$wFixer })
				: moduleBuilder(rootServicesWithoutDeleted)
		}
```

Reply:
or while `createHierarchicalServicesMap`